app/api/campaigns/[campaignId]/withdraw/route.ts

```
@@ -0,0 +1,128 @@
+      where: { campaignId: campaign.id },
+    });
+    const withdrawalsMap: Record<string, number> = {};
+    for (const withdrawal of withdrawals) {
+      if (typeof withdrawalsMap[withdrawal.token] !== 'number') {
+        withdrawalsMap[withdrawal.token] = Number(withdrawal.amount);
+      } else {
+        withdrawalsMap[withdrawal.token] += Number(withdrawal.amount);
+      }
+    }
+    const confirmedPayments =
+      campaign.paymentSummary?.token && token in campaign.paymentSummary?.token
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)